awscli/customizations/configure/sso.py

```diff
@@ -488,6 +488,10 @@ def _prompt_for_cli_output_format(self):
 
     def _prompt_for_profile_config(self, config_name, text, completions=None):
         current_value = self._profile_config.get(config_name)
+
+        if config_name == 'output' and not current_value:
+            current_value = 'json'
```

Comment:
Is this going to persist `output = json` in the profile which we don't do today?

If so, I'd try to avoid a change in behavior with a quick cosmetic change, largely so we don't have to hunt down all of the implications. 
1. If using `current_value` -> `new_value`, is there a way to not persist it if the user just pressed enter to accept the default.
2. Or, can we just communicate the default behavior through the `text` and `prompt_fmt` strings, then still actually collect and "save" `None`?